.circleci/config.yml

```diff
@@ -119,6 +137,9 @@ jobs:
             export NEXT_PUBLIC_OOH_ROTA_TYPE_ID=500
             export NEXT_PUBLIC_OOH_JOB_TYPE_ID=501
             export NEXT_PUBLIC_OOH_RATE=25.00
+            export NEXT_PUBLIC_SENTRY_ENVIRONMENT=production
+            export NEXT_PUBLIC_SENTRY_RELEASE=$CIRCLE_SHA1
+            export NEXT_PUBLIC_SENTRY_DEBUG=false
```

Comment:
This will trigger a source map upload - unsure what will happen when a release goes from development -> staging -> production as they'll have the same sha1 value.